cmd/tealdbg/cdtSession.go

```diff
@@ -169,7 +169,7 @@ func (s *cdtSession) websocketHandler(w http.ResponseWriter, r *http.Request) {
 		state.Update(cdtStateUpdate{
 			dbgState.Stack, dbgState.Scratch,
 			0, 0, "",
-			s.debugger.GetStates(nil),
+			0, s.debugger.GetStates(nil),
```

Comment:
Is the starting opcode budget not available yet for some reason?

Reply:
Good catch, we can just set the starting budget here... I'll push another commit